.gitignore

```diff
@@ -45,3 +45,4 @@ luac.out
 *.hex
 # End of https://www.gitignore.io/api/lua
 valid.txt
+model.t7
```

Comment:
Can you remove this line (and the above one too)?

Reply:
Done!